server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -315,7 +337,13 @@ export class MongoCollection<T> implements core.ICollection<T>, core.IRetryable
 	private sanitizeError(error: any) {
 		if (error) {
 			try {
-				redactJsonKeys(error);
+				Object.keys(error).forEach((key) => {
+					if (typeof error[key] === "object") {
+						this.sanitizeError(error[key]);
+					} else if (!errorResponseKeysAllowList.has(key)) {
+						error[key] = errorSanitizationMessage;
+					}
+				});
 			} catch (err) {
 				Lumberjack.error(`Error sanitization failed.`, undefined, err);
```

Comment:
I hope if sanitization fails, this catch block does not end up logging any cc? :)

Reply:
that is the hope :)